README.md

```diff
@@ -46,5 +46,5 @@ The image will be on the root project.
 We have some on-host tests especially for data structures. Run test with
 
 ```
-cargo test
+cargo test --features "aarch64" -- --test-threads=1 
```

Comment:
Somehow we need to find a way to prevent flakiness on the tests for io related matters